modules.py

```diff
@@ -14,56 +12,39 @@ def obj_terms(x_in, x_name, p_in: dict):
 
     # run each module 
     wave_out = wave_climate(wec,wave_in)
```

Comment:
for these commands you're using (wave_climate and fish_yield_func), are they built into the library? or did you write them somewhere else?